src/main/java/com/dm/dmservicesandbox/domain/UserSignUpRequest.java

```diff
@@ -0,0 +1,73 @@
+package com.dm.dmservicesandbox.domain;
+
+import com.dm.dmservicesandbox.validation.PasswordMatches;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+
+
+@PasswordMatches
+@Getter
+@Setter
+public class UserSignUpRequest {
+
+    private Long userID;
+
+    private String userId;
```

Comment:
Its a bit confusing to have two userId fields, can we name the userId string as userName if thats the intent

Reply:
removed one